inst/@sym/sym.m

```diff
@@ -158,212 +158,224 @@
 %% Author: Colin B. Macdonald
 %% Keywords: symbolic, symbols, CAS
 
-function s = sym(x, varargin)
-
-  if (nargin == 0)
-    s = sym(0);
-    return
+classdef sym < handle
+  properties
```

Comment:
Well, currently only `symsize` is used outside the class. I will check to make sure and then keep others as Protected (just to allow symfun to use them if needed, assuming that derived class can use protected members of the base class in the classdef style too)....